extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/QuarkusHttpHeaders.java

```diff
@@ -0,0 +1,598 @@
+package io.quarkus.vertx.http.runtime;
+
+import static io.netty.handler.codec.http.HttpConstants.COLON;
+import static io.netty.handler.codec.http.HttpConstants.CR;
+import static io.netty.handler.codec.http.HttpConstants.LF;
+import static io.netty.handler.codec.http.HttpConstants.SP;
+import static io.netty.util.AsciiString.CASE_INSENSITIVE_HASHER;
+import static io.netty.util.AsciiString.CASE_SENSITIVE_HASHER;
+
+import java.util.AbstractMap;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.function.Consumer;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufUtil;
+import io.netty.handler.codec.http.HttpHeaders;
+import io.netty.util.AsciiString;
+import io.netty.util.CharsetUtil;
+import io.netty.util.HashingStrategy;
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.impl.HttpUtils;
+
+/**
+ * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
```

Comment:
Looks like formatting failed, maybe when fixing it remove my accidental copy paste of the author tag? Other than that LGTM

Reply:
`@author` tags should be removed. See https://github.com/quarkusio/quarkus/blob/master/CONTRIBUTING.md#coding-guidelines